pirate/pirate.py

```diff
@@ -146,6 +149,9 @@ def parse_args(args_in):
                         default=1, type=int,
                         help='the number of pages to fetch. '
                              '(only used with --recent)')
+    parser.add_argument('-r', '--total-results',
+                        type=int,
+                        help='The number of links to return after fetching all results.')
```

Comment:
Shouldn't we use the same description here?

Reply:
Whoops, yep, good catch. I've updated it